test/integration/smoke/test_ssvm.py

```diff
@@ -1197,3 +1205,148 @@ def test_10_destroy_cpvm(self):
+        """Test NFS Version on Secondary Storage mounted properly on SSVM
+        """
+
+	# 1) List SSVM in zone
+	# 2) Get id and url from mounted nfs store
+	# 3) Update NFS version for previous image store
+	# 4) Stop SSVM
+	# 5) Check NFS version of mounted nfs store after SSVM starts 
+
+	nfs_version = self.config.nfsVersion
+	if nfs_version == None:
+	    self.skipTest('No NFS version provided in test data')
+
+	#List SSVM for zone id
+        list_ssvm_response = list_ssvms(
+            self.apiclient,
+            systemvmtype='secondarystoragevm',
+            state='Running',
+            zoneid=self.zone.id
+        )
+        self.assertEqual(
+            isinstance(list_ssvm_response, list),
+            True,
+            "Check list response returns a valid list"
+        )
+        self.assertEqual(
+            len(list_ssvm_response),
+            1,
+            "Check list System VMs response"
+        )
+
+        ssvm = list_ssvm_response[0]
+        image_stores_response = ImageStore.list(self.apiclient,zoneid=self.zone.id)
+
+	if self.hypervisor.lower() in ('vmware', 'hyperv'):
+            # SSH into SSVMs is done via management server for Vmware and Hyper-V
+            result = get_process_status(
+                self.apiclient.connection.mgtSvr,
+                22,
+                self.apiclient.connection.user,
+                self.apiclient.connection.passwd,
+                ssvm.privateip,
+                "mount | grep 'type nfs'",
+                hypervisor=self.hypervisor)
+
+	for res in result:
+            split_res = res.split(" on ")
+            mounted_img_store_url = split_res[0]
+            for img_store in image_stores_response:
+		img_store_url = str(img_store.url)
+		if img_store_url.startswith("nfs://"):
+		    img_store_url = img_store_url[6:]
+                    #Add colon after ip address to match output from mount command
+                    first_slash = img_store_url.find('/')
+                    img_store_url = img_store_url[0:first_slash] + ':' + img_store_url[first_slash:]
+                    if img_store_url == mounted_img_store_url:
+                        img_store_id = img_store.id
+                        break
+
+        self.assertNotEqual(
+            img_store_id,
+            None,
+            "Check image store id mounted on SSVM"
+        )
+
+	#Update NFS version for image store mounted on SSVM
+        updateConfigurationCmd = updateConfiguration.updateConfigurationCmd()
+        updateConfigurationCmd.name = "secstorage.nfs.version"
+        updateConfigurationCmd.value = nfs_version
+        updateConfigurationCmd.imagestoreuuid = img_store_id
+
+        updateConfigurationResponse = self.apiclient.updateConfiguration(updateConfigurationCmd)
+        self.logger.debug("updated the parameter %s with value %s"%(updateConfigurationResponse.name, updateConfigurationResponse.value))
+
+	#Stop SSVM
+	self.debug("Stopping SSVM: %s" % ssvm.id)
```

Comment:
Please consider changing the log level to `INFO` as this information seems like it would be useful for monitoring the progress of the test.